test/Microsoft.DotNet.Tools.Tests.Utilities/NuGet.Config

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why is Utilities depending on the Test Packages? I am not sure this is a good thing.